packages/mobile/scripts/ios-simulator.js

```
@@ -0,0 +1,279 @@
+  const manager = new IOSSimulatorManager();
+  const command = process.argv[2];
+  
+  try {
+    switch (command) {
+      case 'setup':
+        await manager.setupSimulator();
+        break;
+        
+      case 'reset':
+        const bootedSim = await manager.findBootedSimulator();
+        if (bootedSim) {
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)